packages/trpc/server/routers/loggedInViewer/me.handler.ts

```diff
@@ -37,5 +37,6 @@ export const meHandler = async ({ ctx }: MeOptions) => {
     theme: user.theme,
     hideBranding: user.hideBranding,
     metadata: user.metadata,
+    allowDynamicBooking: user.allowDynamicBooking,
```

Comment:
We don't really need to pull it here as well. Pulling it in the sessionMiddleware alone should suffice.

Reply:
completed the change